service/prometheus/scrapeconfig.go

```diff
@@ -0,0 +1,51 @@
+package prometheus
+
+import (
+	"fmt"
+
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/config"
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+// GetTarget takes a Kubernetes Service, and returns a LabelSet,
+// suitable for use as a target.
+func GetTarget(service v1.Service) model.LabelSet {
+	target := model.LabelSet{
+		model.LabelName(
+			fmt.Sprintf("%s.%s", service.Name, service.Namespace),
```

Comment:
I reckon it's probably more efficient to use the service's endpoints as targets directly, but that feels premature to me.